test/statetest/statetest_loader.cpp

```diff
@@ -182,23 +182,49 @@ evmc_revision to_rev(std::string_view s)
     throw std::invalid_argument{"unknown revision: " + std::string{s}};
 }
 
-static void from_json(const json::json& j, TestMultiTransaction& o)
+/// Load common parts of Transaction or TestMultiTransaction.
+static void from_json_tx_common(const json::json& j, state::Transaction& o)
 {
-    if (j.contains("gasPrice"))
+    o.sender = from_json<evmc::address>(j.at("sender"));
+
+    if (const auto& to = j.at("to"); !to.get<std::string>().empty())
+        o.to = from_json<evmc::address>(to);
+
+    if (const auto gas_price_it = j.find("gasPrice"); gas_price_it != j.end())
     {
         o.kind = state::Transaction::Kind::legacy;
-        o.max_gas_price = from_json<intx::uint256>(j.at("gasPrice"));
+        o.max_gas_price = from_json<intx::uint256>(*gas_price_it);
         o.max_priority_gas_price = o.max_gas_price;
+        if (j.contains("maxFeePerGas") || j.contains("maxPriorityFeePerGas"))
+            throw std::invalid_argument(
```

Comment:
This could be in braces {}, because multi-line.

Reply:
Changed.